src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -318,6 +316,7 @@ private void processFiles(List<File> files) throws CheckstyleException {
      * @param file a file to process.
      * @return a sorted set of violations to be logged.
      * @throws CheckstyleException if error condition within Checkstyle occurs.
+     * @return The set of violations found
```

Comment:
There is already a return tag 2 lines up, so this change isn't right.

Reply:
Pfft. This is a leftover from some refactor that I late undid.

Reply:
We have a Javadoc linter in CI. It should be configured to fail on warnings.